stories/Components/Forms/Textpanel/FormTypes.jsx

```diff
@@ -145,35 +148,37 @@ export const FormTypes = ({
         </div>
       )}
       {args.variant === "stacked form" && (
-        <div className="grid-x grid-margin-x grid-margin-y stacked-form">
-          <div className="large-4 medium-6 cell">
-            <div className="grid-x">
-              <div className="large-12 cell">
-                <Heading type="2" label={anchorTag} />
-                <p>{para}</p>
-                <div className="grid-x">
-                  <div className="cell">
-                    <Inputcomponent
-                      element={element}
-                      type={type}
-                      placeholder={placeholder}
-                      required={required}
-                      mode={mode}
-                    />
-                  </div>
-                  <div className="cell">
-                    <Inputcomponent
-                      element={element}
-                      type={type}
-                      placeholder={placeholderemail}
-                      required={required}
-                      mode={mode}
-                    />
-                  </div>
-                  <div className="cell">
-                    <button type="submit" className="button button-primary">
-                      {button}
-                    </button>
+        <div className='stacked-form'>
```

Comment:
@Ashish-srij Remove `stacked-form` class if not being used as it adds to unnecessary extra divs. Else use it in the existing div structure.

Reply:
@shishirphadke Div removed and class moved to bottom div